src/app/organizations/collections/remove-collection/remove-collection.component.html

```diff
@@ -0,0 +1,14 @@
+<h1 mat-dialog-title>Remove Collection</h1>
+<div mat-dialog-content>
+  <app-alert></app-alert>
+  <p>
+    Are you sure you want to <strong>remove</strong> the collection <strong>{{data.collectionName}}</strong>?
+  </p>
+  <p>
+    <strong>The collection will be deleted, and you will not be able to restore it.</strong>
+  </p>
+</div>
+<div mat-dialog-actions align="end">
+  <button mat-button mat-dialog-close cdkFocusInitial matTooltip="Cancel" data-cy="cancel-remove-collection">Cancel</button>
+  <button mat-flat-button [disabled]="removeClicked" (click)="removeCollection(); removeClicked = true" matTooltip="Remove the collection" data-cy="accept-remove-collection">Remove</button>
```

Comment:
we usually tie the [disabled] to some sort of loading state (alertservice or akita store) to avoid complications with the user performing multiple requests